src/mmw/js/src/analyze/views.js

```diff
@@ -222,16 +228,25 @@ var TabContentsView = Marionette.CollectionView.extend({
 var TableRowView = Marionette.ItemView.extend({
     tagName: 'tr',
     template: tableRowTmpl,
-        templateHelpers: function() {
+    templateHelpers: function() {
+        var quotient = this.options.kiloMeters ? M_IN_KM : 1;
+
         return {
             // Convert coverage to percentage for display.
-            coveragePct: (this.model.get('coverage') * 100).toFixed(1)
+            coveragePct: (this.model.get('coverage') * 100).toFixed(1),
```

Comment:
This function seems to make sure that there is only one digit after the decimal point.  There does not seem to be a similar function among the default supplied by nunjucks, is there an economical way to create a custom filter to do this?